airbyte-config/init/src/main/java/io/airbyte/config/init/LocalDefinitionsProvider.java

```diff
@@ -28,41 +21,48 @@
 import java.util.stream.Collectors;
 
 /**
- * This provider contains all definitions according to the local yaml files.
+ * This provider contains all definitions according to the local catalog json files.
  */
 final public class LocalDefinitionsProvider implements DefinitionsProvider {
 
-  public static final Class<?> DEFAULT_SEED_DEFINITION_RESOURCE_CLASS = SeedType.class;
-
-  private final static String PROTOCOL_VERSION = "protocol_version";
-  private final static String SPEC = "spec";
+  private static final String LOCAL_CONNECTOR_CATALOG_PATH = CatalogDefinitionsConfig.getLocalConnectorCatalogPath();
 
-  private Map<UUID, StandardSourceDefinition> sourceDefinitions;
-  private Map<UUID, StandardDestinationDefinition> destinationDefinitions;
+  public CombinedConnectorCatalog getLocalDefinitionCatalog() {
```

Comment:
Honest question because I have no idea - Are these new methods covered by existing tests?

Reply:
Good question Joe! They are covered by the previous LocalDefinitionsProvider tests! 

The reason this is so seemless is the parsing logic was taken from RemoteDefinitionsProvider, and both LocalDefinitionsProvider and RemoteDefinitionsProvider implement the same interface